packages/babel-preset-default/test/index.js

```diff
@@ -16,6 +16,7 @@ describe( 'Babel preset default', () => {
 
 		const output = transform( input, {
 			configFile: false,
+			envName: 'production',
```

Comment:
With this change, we test again production configuration rather than test env ... It is more helpful now :)